pkgs/applications/misc/feedbackd/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv
+, fetchFromGitLab
+, meson
+, ninja
+, pkg-config
+, cmake
+, wrapGAppsHook
+, glib
+, gsound
+, libgudev
+, json-glib
+, vala
+, gobject-introspection
+}:
+
+stdenv.mkDerivation rec {
+  pname = "feedbackd";
+  version = "0.0.0+git20200527";
+
+  src = fetchFromGitLab {
+    domain = "source.puri.sm";
+    owner = "Librem5";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1wbkzxnqjydfgjvp7vz4ghczcz740zcb1yn90cb6gb5md4n6qx2y";
+  };
```

Comment:
when I tried building this I was having an issue downloading this archive 
```
wget "https://source.puri.sm/api/v4/projects/Librem5%2Ffeedbackd/repository/archive.tar.gz?sha=v0.0.0+git20200527"                                
https://source.puri.sm/api/v4/projects/Librem5%2Ffeedbackd/repository/archive.tar.gz?sha=v0.0.0+git20200527
Resolving source.puri.sm (source.puri.sm)... 178.63.173.204
Connecting to source.puri.sm (source.puri.sm)|178.63.173.204|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
ERROR 404: Not Found.
```

Reply:
Did you build this on nixos 20.03? There was a bug in `fetchFromGitLab` and wasn't encoding the version. This was fixed here https://github.com/NixOS/nixpkgs/commit/9b7a4694a0d9ca9717a5bbdb2a1e9b3af1f7fd1f